openfl-tutorials/experimental/workflow/CrowdGuard/PoisoningAttackDemo.ipynb

```diff
@@ -430,9 +430,8 @@
     "        state_dicts = [model.state_dict() for model in models]\n",
     "        state_dict = new_model.state_dict()\n",
     "        for key in models[1].state_dict():\n",
-    "            state_dict[key] = np.sum(\n",
```

Comment:
can you please update the similar change in PoisoningAttackDemoReduced.ipynb?

Reply:
I couldn't find any instances of those issues in `PoisoningAttackDemoReduced.ipynb`, but maybe I'm missing something. Could you point me to the lines of code you're referring to?

Reply:
i tested it again with fresh clone with new python virtual env, i see that no change is required in PoisoningAttackDemoReduced.ipynb, please ignore this comment.